HPS/Heartland/view/frontend/templates/paypal/review/orderreview.phtml

```diff
@@ -0,0 +1,124 @@
+<?php
+$lid = $this->getOrderID();
+		 
+$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+$order = $objectManager->create('Magento\Sales\Model\Order')->load($lid);
+//echo '<pre>'; print_r($order->debug()); echo '</pre>';
+
+// Details
+$shippingAddress = $order->getShippingAddress();
+$billingAddress = $order->getBillingAddress(); 
+?>
+<div class="container">
+  <div class="row">
+    <div class="col-md-11">
+      <h1>Order details:</h1>
+    </div>
+  </div>
+
+  <hr>
+  <div class="row">
+    <div class="col-md-6 info" style="float:left;width:50%">
+      <strong>Billing Info:</strong><br>
+	  <?php
+		echo $billingAddress->getFirstname().' '.$billingAddress->getLastname().'</br>';
+		echo $billingAddress->getStreetLine(1);
+			if($billingAddress->getStreetLine(2)) { 
+			echo ', ' . $billingAddress->getStreetLine(2);
+			}
+		echo '</br>';
+		echo $billingAddress->getCity().', '.$billingAddress->getRegion().'</br>';
+		echo $billingAddress->getCountry_id().' '.$billingAddress->getPostcode().'</br>'; 
+		echo 'T: '.$billingAddress->getTelephone();  
+	  ?>          
+    </div>
+    <div class="col-md-6 info">
+      <strong>Shipping Info:</strong><br>
+	  <?php
+		echo $shippingAddress->getFirstname().' '.$shippingAddress->getLastname().'</br>';
+		echo $shippingAddress->getStreetLine(1);
+			if($shippingAddress->getStreetLine(2)) { 
+			echo ' , ' . $shippingAddress->getStreetLine(2);
+			}
+		echo '</br>';
+		echo $shippingAddress->getCity().', '.$shippingAddress->getRegion().'</br>';
+		echo $shippingAddress->getCountry_id().' '.$shippingAddress->getPostcode().'</br>'; 
+		echo 'T: '.$shippingAddress->getTelephone().'</br></br>';   
+	  ?>        
+    </div>
+  </div>
+
+  <hr>
+  <table class="table">
+      <thead>
+          <tr class="active">
+              <th width="10%">Name</th>
+              <th width="60%">SKU</th>
+              <th width="10%">Amount</th>
+              <th width="10%">Quantity</th>
+              <th>Subtotal</th>
+          </tr>
+      </thead>
+      <tbody>
+          <?php 			     
+			// Product Items
+			$items = $order->getAllItems();
+			foreach($items as $i):			
+				$_product = $objectManager->create('Magento\Catalog\Model\Product')->load($i->getProductId());	
+			//	echo '<pre>'; print_r($_product->debug()); echo '</pre>';
+					$_productName = $_product->getName();
+					$_productSku = $_product->getSku();						
+					$_productPrice = $_product->getPrice();							
+					$_productQty = intval($i->getQtyOrdered());		
+					
+				echo "<tr>";
```

Comment:
avoid echo to output html code. use echo only for the PHP variables

Reply:
Fixed